src/Evolution/Systems/Cce/Components/KleinGordonCharacteristicEvolution.hpp

```diff
@@ -0,0 +1,78 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+#include "Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp"
+#include "Evolution/Systems/Cce/KleinGordonSystem.hpp"
```

Comment:
```cpp
#include "Parallel/GlobalCache.hpp"
#include "Parallel/Local.hpp"
#include "Parallel/Phase.hpp"
#include "Time/Actions/SelfStartActions.hpp"
#include "Utilities/TMPL.hpp"
```

Reply:
Out of curiosity, these header files are already included in `Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp`. I think it should be fine to omit them here, don't you think?

Reply:
Technically, yes, it's ok to omit them here because they are included in the other `hpp`. However, if somebody in the future decided to remove one of them from `CharacteristicEvolution.hpp`, then this file would fail to compile and somebody would have to spend time trying to figure out why.

So as a rule of thumb, we just always include all files that we need. Even if they may have been included via another file.